PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please add a line in the relevant section of [CHANGELOG.md](https://github.com/ImperialCollegeLondon/virtual_rainforest/CHANGELOG.md) to document the change (include PR #) - note reverse order of PR #s. If necessary, also add to the list of breaking changes.
+
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Optimization (back-end change that speeds up the code)
+- [ ] Bug fix (non-breaking change which fixes an issue)
+
+
+# Key checklist:
+
+- [ ] No style issues: `$ flake8`
```

Comment:
Probably we want to ask if all `pre-commit` hooks pass, not just `flake8`. If `pre-commit` is properly setup and the changes were made locally, this is automatically fulfilled but not if they changes were made directly in GitHub. So, something along these lines will be better:

```suggestion
- [ ] No style issues: `$ poetry run pre-commit run --all-files`
```